packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -189,9 +189,10 @@ class AndroidWorkflow extends DoctorValidator implements Workflow {
         <String>[sdkManagerPath, '--licenses'],
         environment: sdkManagerEnv,
     );
-    stdout.addStream(process.stdout);
-    stderr.addStream(process.stderr);
-    process.stdin.addStream(stdin);
+
+    await stdout.addStream(process.stdout);
+    await stderr.addStream(process.stderr);
+    await process.stdin.addStream(stdin);
```

Comment:
This will cause a hang. You're waiting for the stdout stream to close before you add the stderr stream, then waiting for that to close before you add the stdin stream. The stdout and stderr streams presumably won't close at all until they get input from stdin.